_posts/2019-02-25-text-layout-kickoff.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title:  "Low level text layout kickoff"
+date:   2019-02-27 11:16:42 -0700
+categories: [rust, skribo, text]
+---
+This post is to announce a new Rust library for low-level text layout, called "[skribo]" (the Esperanto word for "writing"). This has been a major gap in the Rust ecosystem, and I hope the new crate can improve text handling across the board.
+
+## Multiple shaping engines
+
+The skribo library doesn't do shaping itself; it is basically a glue library on top of an actual shaping engine. Most users will use [HarfBuzz] for this, the industry-standard library. However, a goal for skribo is that it abstracts over the details of the shaping engine. Other choices include platform text (DirectWrite on Windows, CoreText on macOS and iOS). Also, if and when a Rust-native shaping library emerges, we want to make it easy to switch to that.
```

Comment:
I would briefly explain what shaping is before diving in here?

Reply:
Better I think is to give a visual example of with/without. I'm making that now.